server/status.go

```diff
@@ -254,7 +254,7 @@ func (s *statusServer) handleDetailsLocal(w http.ResponseWriter, r *http.Request
 		BuildInfo: util.GetBuildInfo(),
 	}
 	if addr, err := s.gossip.GetNodeIDAddress(s.gossip.GetNodeID()); err == nil {
-		local.Address = addr.(util.UnresolvedAddr)
+		local.Address = *addr.(*util.UnresolvedAddr)
```

Comment:
hm. `gossip.GetNodeIDAddress` should return a `util.UnresolvedAddr` - that's the underlying type.